src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html

```diff
@@ -15,7 +15,7 @@
                     <legend style="font-size: 18px;">{{tab.alias}}</legend>
 
                     <umb-property property="property"
-                                  ng-repeat="property in tab.properties">
+                                  ng-repeat="property in tab.properties | orderBy:'+sortOrder'">
```

Comment:
I seem to recall that the OrderBy filter isn't that performant. I'd be tempted to sort them using in the controller before they are bound.